apps/api_web/lib/api_web/controllers/stop_controller.ex

```diff
@@ -174,7 +179,11 @@ defmodule ApiWeb.StopController do
     #{swagger_path_description("/data")}
     """)
 
-    include_parameters(@includes)
+    include_parameters(@includes,
```

Comment:
For `show` it's not possible to include a route: we shouldn't include `route` as one of the parameters here.